mlflow/types/llm.py

```diff
@@ -276,25 +276,29 @@ class ChatResponse(_BaseDataclass):
 
     Args:
         id (str): The ID of the response.
-        object (str): The object type.
-        created (int): The time the response was created.
-            **Optional**, defaults to the current time.
         model (str): The name of the model used.
         choices (List[:py:class:`ChatChoice`]): A list of :py:class:`ChatChoice` objects
             containing the generated responses
         usage (:py:class:`TokenUsageStats`): An object describing the tokens used by the request.
+        _object (str): The object type. The value should always be 'chat.completion'
```

Comment:
The docstring should still point to `object` property and mention that you cannot set any value than `"chat.completion"`. Ideally we shouldn't surface the private field to users at all, as if there is only the `object` field being immutable🙂

Reply:
got it